### PR TITLE
Add statsd guide

### PR DESCRIPTION
**Pull request by OdysLam**
##### Summary

Add the guide for adding any StatsD data source.

##### Component Name

`docs/guides/monitor/statsd.md`

### CODE REVIEW

File: docs/guides/monitor/statsd.md [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+- Place the configuration file inside `netdata/collectors/statsd.plugin`
+- Add a reference in `netdata/collectors/statsd.plugin/Makefile.am`. For example, if we contribute the `k6.conf` file:
+```Makefile
+dist_statsdconfig_DATA = \
+    example.conf \
+    k6.conf \
+    $(NULL)
+```
+
+## ???
```
Comment (ilyam8) [inline docs/guides/monitor/statsd.md:275]: Is `???` a placeholder for the title and it supposed to be the title? 😄

File: docs/guides/monitor/statsd.md [hunk 2]
```diff
@@ -0,0 +1,292 @@
+<!--
+title: How to use any StatsD data source with Netdata
+description: "How to create a new collector for any applications that uses StatsD "
+image: /img/seo/guides/monitor/process.png
+custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/guides/monitor/statsd.md
+-->
```
Comment (joelhans) [inline docs/guides/monitor/statsd.md:6]:
```suggestion
<!--
title: How to use any StatsD data source with Netdata
description: "Learn how to monitor any custom application instrumented with StatsD with per-second metrics and fully customizable, interactive charts."
image: /img/seo/guides/monitor/statsd.png
author: "Odysseas Lamtzidis"
author_title: "Developer Advocate"
author_img: "/img/authors/odysseas-lamtzidis.jpg"
custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/guides/monitor/statsd.md
-->
```

**Review by joelhans — APPROVED** [review]

**Review by ilyam8 — APPROVED** [review]

File: docs/guides/monitor/statsd.md [hunk 3]
```diff
@@ -0,0 +1,297 @@
...
+    type = stacked
+```
+
+> Take note that Netdata will report the rate for metrics and counters, even if k6 or another application sends an _absolute_ number. For example, k6 sends absolute HTTP requests with `http_reqs`, but Netdat visualizes that in `requests/second`.
+
+To enable this StatsD configuration, [restart Netdata](/docs/configure/start-stop-restart.md).
+
+## Final touches
+
+At this point, you have used StatsD to gather metrics for k6, creating a whole new section in your Netdata dashboard in the process. Uil can further customize the icon of the particular section, as well as the description for each chart.
```
Comment (jsoref) [inline docs/guides/monitor/statsd.md:218]: What's `Uil`?

Comment (ilyam8) [reply docs/guides/monitor/statsd.md:218]: nice catch @jsoref !

Comment (OdysLam) [reply docs/guides/monitor/statsd.md:218]: Uil is actually an [institute](https://uil.unesco.org/unesco-institute)!

But thanks for the catch @jsoref 😅

Comment (jsoref) [reply docs/guides/monitor/statsd.md:218]: Fwiw, that's from https://github.com/check-spelling/netdata/commit/cd752b3b3057f5f54171cd7525737e1a925b32f3#commitcomment-49502749

I'll send a PR for everything else. But this one I can't figure out at all.

If you're up for it, I think netdata would benefit from running the check-spelling action 🙂 .

Comment (OdysLam) [reply docs/guides/monitor/statsd.md:218]: > But this one I can't figure out at all .

Yeah, we know @joelhans  :P . Thanks, in any case, we will take it from here.

As fas the GHA, I think @joelhans  @Ferroin have chatted about this. It's seems nice to have this for good measure.

Comment (jsoref) [reply docs/guides/monitor/statsd.md:218]: Ah, so it should be `UIL` per their page:
> UNESCO Institute for Lifelong Learning (UIL)

I can toss that in w/ everything else :-).

**Pull request merged**